docs/specifications/tests/DNSSEC-TP/dnssec02.md

```diff
@@ -147,11 +152,13 @@ queries follow, unless otherwise specified below, what is specified for
        5. The AA flag is not set in the [DNSSEC Response].
        6. There is no DNSKEY record with matching owner name in the answer
           section of the [DNSSEC Response].
-    3. Retrieve the DNSKEY RRset (store as "DNSKEY RRs") from the
+    3. Add the name server IP address to the *Responding Child Name Servers* set.
+    4. Retrieve the DNSKEY RRset (store as "DNSKEY RRs") from the
        [DNSSEC Response].
-    4. Retrieve the RRSIG records covering the DNSKEY RRset, possibly
+    5. Retrieve the RRSIG records covering the DNSKEY RRset, possibly
        none (store as "DNSKEY RRSIG") from the [DNSSEC Response].
-    5. For each DS in *DS Records*, do:
+    6. Empty the *DNSKEY Match DS* set.
```

Comment:
This isn't needed as the set is never used before.

Reply:
It must be empty each time a new name server is inspected. It is defined as a "global variable" but it is used as a "local variable".

Reply:
I think that is problematic. Because this set is also used in step 14, which is supposed to contain data for all name servers. Right now it will contain data for only one name server, namely the last in the *Child NS IP* set.

Reply:
I will investigate.